app/code/Magento/Checkout/view/frontend/templates/total/default.phtml

```diff
@@ -4,7 +4,10 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Checkout\Block\Total\DefaultTotal */
+/**
+ * @var \Magento\Checkout\Block\Total\DefaultTotal $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
 ?>
 
```

Comment:
Please remove here not needed closing and opening PHP tags and combine PHP doc blocks to single block.

Reply:
fixed